packages/renderer/src/lib/table/Table.svelte

```diff
@@ -0,0 +1,161 @@
+<style>
+.grid-table {
+  display: grid;
+}
+</style>
+
+<script lang="ts">
+/* eslint-disable import/no-duplicates */
+// https://github.com/import-js/eslint-plugin-import/issues/1479
+import { afterUpdate, tick } from 'svelte';
+import Checkbox from '../ui/Checkbox.svelte';
+import type { Column, Row } from './table';
+import { flip } from 'svelte/animate';
+/* eslint-enable import/no-duplicates */
+
+export let kind: string;
+export let data: any[];
+export let columns: Column<any>[];
+export let row: Row<any>;
+
+// number of selected items in the list
+export let selectedItemsNumber: number = 0;
+$: selectedItemsNumber = row.info.selectable
+  ? data.filter(object => row.info.selectable?.(object)).filter(object => object.selected).length
```

Comment:
any reason why you don't have ` data.filter(object => row.info.selectable?.(object) && object.selected)`

Reply:
None at all, I just copied from ImagesList and VolumesList and didn't think to fix. :) I'll do it in follow-up PR.